models/migrations/migrations.go

```diff
@@ -849,6 +742,19 @@ func generateOrgRandsAndSalt(x *xorm.Engine) (err error) {
 	return sess.Commit()
 }
 
+// Enumerate all the unit types as of v16 (Gitea 1.1.0)
+const (
+	V16UnitTypeCode            = iota + 1 // 1 code
+	V16UnitTypeIssues                     // 2 issues
+	V16UnitTypePRs                        // 3 PRs
+	V16UnitTypeCommits                    // 4 Commits
+	V16UnitTypeReleases                   // 5 Releases
+	V16UnitTypeWiki                       // 6 Wiki
+	V16UnitTypeSettings                   // 7 Settings
+	V16UnitTypeExternalWiki               // 8 ExternalWiki
+	V16UnitTypeExternalTracker            // 9 ExternalTracker
+)
+
 // TAction defines the struct for migrating table action
 type TAction struct {
```

Comment:
this type and mostly all other below are then not used anymore. Also as functions below `fixLocaleFileLoadPanic`, `trimCommitActionAppURLPrefix`, `issueToIssueLabel`, `attachmentRefactor`, `renamePullRequestFields`, `cleanUpMigrateRepoInfo`, `generateOrgRandsAndSalt`, `convertDateToUnix`, `ldapUseSSLToSecurityProtocol` and might be also others in this file.

Reply:
@lafriks  Done. Removed and checked unused functions with [staticcheck](https://staticcheck.io/docs/). `v16UnitTypeIssues` was still being used by v70.go and `v16UnitTypeCode` by v76.go, so I've moved the const declarations there.